packages/stats/src/types.ts

```diff
@@ -0,0 +1,11 @@
+// common types
+
+import {ToolShape} from '@tracespace/parser'
+
+export interface Stats {
+  tools: Map<string, ToolShape>
+  drillsPerTool: Map<string, number>
+  totalDrills: number
+  minDrillSize: number
+  maxDrillSize: number
+}
```

Comment:
I'm a little old-fashioned, but I don't really love the idea using `Map`'s as part of the public interface vs. a plain array or object. Maybe something closer to this instead?

```ts
interface DrillUsage {
  diameter: number,
  count: number,
}

interface Stats {
  drillHits: DrillUsage[]
  drillRoutes: DrillUsage[]
  // ...
}
```